src/main/fc/fc_tasks.c

```diff
@@ -106,7 +106,7 @@ static void taskUpdateBattery(uint32_t currentTime)
 {
 #ifdef USE_ADC
```

Comment:
Not part of this PR, but we should probably check for `USE_ESC_SENSOR` here as well, otherwise the ESC sensor functionality will not be available on boards that don't use the ADC, where they would be most welcome.